docs/docs/concepts/training-data-importers.mdx

```diff
@@ -1,220 +0,0 @@
----
-id: training-data-importers
```

Comment:
isn't this still relevant? flows can be seen as training data, no?

Reply:
I will poll CSE but I don't think anyone uses this since 3.0 where you can already arbitrarily split everything up into folders / files. We should probably deprecate it.

Reply:
I think I made this recommendation as well

Reply:
I'm gonna move it into the spaces doc instead for now.